LICENSE.txt

```diff
@@ -1,15 +1,14 @@
-/* Copyright (c) 2018-Present Expedia Group.
- * All rights reserved.  http://www.homeaway.com
-
+/* Copyright (C) 2018-2019 Expedia, Inc.
```

Comment:
Ah, this is correct, so maybe just an issue with that .properties file above? I assume that one is done manually not be the plugin or something.

Reply:
yep